go-controller/pkg/libovsdbops/qos.go

```diff
@@ -105,3 +105,23 @@ func RemoveQoSesFromLogicalSwitchOps(nbClient libovsdbclient.Client, ops []libov
 	modelClient := newModelClient(nbClient)
 	return modelClient.DeleteOps(ops, opModels)
 }
+
+func RemoveQoSsFromLogicalSwitchesWithPredicateOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation,
```

Comment:
This is just the commit with all the new libovdbops functions, that are used later. I thought it may be easier to review them separately, but I can squash.